### DISCUSSION TITLE
Auto-complete just writes on begin -- possible bug

### DISCUSSION THREAD

**Discussion by muxina-credo** [texstudio-org/texstudio#2076] [Q&A] [answered]
I have updated TexStudio from 4.1.2 to 4.2. When I type \begin the auto-complete immediately prompts me \begin{*environment-name*} as normal. Usually I could press ENTER and start naming the environment that what I wrote inside curly brackets in \begin{} would be copied to \end{}. Now this doesn't happen.

I don't know if this is an actual bug or maybe some default change in configurations due to the latest update.

Thank you in advance for any help given.

EDIT:

This is a little strange but if I wait two seconds before starting to type the normal behaviour resumes. But if I do it immediately after pressing ENTER the behaviour mention above happens.

**Comment by sunderme** [comment, marked as answer]
this has been fixed in the dev build.
Will be fixed in 4.2.1
